src/clothing_mod.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#ifndef CLOTHING_MOD_H
+#define CLOTHING_MOD_H
+
+#include <cstddef>
+#include <string>
+#include <vector>
+
+#include "type_id.h"
+
+class JsonObject;
+class player;
+class item;
+
+enum cm_type : int {
```

Comment:
Need to expand the name, cm_type doesn't provide enough to go on.
a full clothing_mod_type would be fine.